### ISSUE TITLE
errors on load

### ISSUE THREAD

**Issue by eli-damon** [fluffware/export_layer_combinations#2]
export_layer_combinations is exactly what I need for rendering game cards, but it seems to contain errors that prevent inkscape from loading it (attached).

[inkscape errors.txt](https://github.com/fluffware/export_layer_combinations/files/9540171/inkscape.errors.txt)

**Comment by fluffware** [comment]
It looks like export_layer_combinations.inx is corrupted (it looks like HTML). How did you download the files? Unless you use a Git-client, the easiest way is probably to use "Code" button and then "Download ZIP".

**Comment by eli-damon** [comment]
Thanks for responding. I'm sorry, I did something very silly. I downloaded the download page instead of the actual files. But I download the actual files, and that didn't help. The extension still doesn't appear in the extensions menu after loading Inkscape. Attached are the error messages.

[export_layer_combinations.txt](https://github.com/fluffware/export_layer_combinations/files/9543151/export_layer_combinations.txt)
